dataflow/tests/java22/unnamed-pattern/Expected.txt

```diff
@@ -0,0 +1,90 @@
+2 -> 3 EACH_TO_EACH
+3 -> 4 EACH_TO_EACH
+4 -> 8 THEN_TO_BOTH
+4 -> 10 ELSE_TO_BOTH
+8 -> 12 EACH_TO_EACH
+10 -> 12 EACH_TO_EACH
+12 -> 0 EACH_TO_EACH
+
+2:
+Process order: 1
+TransferInput#0
+Before:   reaching definitions = none
+~~~~~~~~~
+<entry>
+
+3:
+Process order: 2
+TransferInput#1
+Before:   reaching definitions = none
+~~~~~~~~~
+x   [ VariableDeclaration ]
+0   [ IntegerLiteral ]
+x = 0   [ Assignment ]
+switch#num0   [ VariableDeclaration ]
+switch#num0   [ LocalVariable ]
+i   [ LocalVariable ]
+switch#num0 = i   [ Assignment ]
+marker (start of switch statement #0)   [ Marker ]
+org.checkerframework.dataflow.cfg.node.AnyPatternNode@6743e411   [ AnyPattern ]
+org.checkerframework.dataflow.cfg.node.DeconstructorPatternNode@3eb25e1a   [ DeconstructorPattern ]
+case org.checkerframework.dataflow.cfg.node.DeconstructorPatternNode@3eb25e1a:   [ Case ]
```

Comment:
@smillst one concern here.  Even though the source code has two `case` clauses, I only see one `case` here in the CFG test output.  Does it look right to you?  I do see the statements from the other case.  But maybe it gets treated as a "default" so the second `DeconstructorPatternNode` gets trimmed.

Reply:
If you switch the test case to use named patterns the output is the same (plus additional nodes for the named local variable.). So I think this is ok.